palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/B32114928.output

```diff
@@ -1,6 +1,7 @@
 class B32114928 {
     {
-        Class<T> tClass = (Class<T>) verifyNotNull((ParameterizedType) getClass().getGenericSuperclass())
-                .getActualTypeArguments()[0];
+        Class<T> tClass = (Class<T>)
+                verifyNotNull((ParameterizedType) getClass().getGenericSuperclass())
+                        .getActualTypeArguments()[0];
```

Comment:
Unclear if we want to be so strict with all methods, or if we should exempt non-side-effect-sounding methods like those that start with `get[A-Z0-9]`. @iamdanfox thoughts?